src/components/DistanceRequest.js

```diff
@@ -84,7 +102,7 @@ function DistanceRequest({transactionID, transaction, translate}) {
     return (
         <>
             <View
-                style={{maxHeight: scrollContainerMaxHeight}}
+                style={{maxHeight: scrollContainerMaxHeight, minHeight: variables.baseMenuItemHeight, flexShrink: 2}}
```

Comment:
These styles need to be moved to the styles file.

Reply:
moved 👍